core/domain/wipeout_service_test.py

```diff
@@ -1225,6 +1225,23 @@ def test_one_config_property_is_pseudonymized(self) -> None:
         self.assertEqual(
             metadata_model.committer_id, config_mappings[self.CONFIG_1_ID])
 
+    def test_remove_user_from_user_groups(self) -> None:
+        user_group_model = user_models.UserGroupModel(
+            id='group_1',
+            name='Test Group',
+            user_ids=[self.user_1_id, self.user_2_id]
+        )
+        user_group_model.put()
+
+        user_group = user_models.UserGroupModel.get_by_id('group_1')
+        self.assertIn(self.user_1_id, user_group.user_ids)
+
+        wipeout_service.delete_user(
+            wipeout_service.get_pending_deletion_request(self.user_1_id))
+
+        updated_user_group = user_models.UserGroupModel.get_by_id('group_1')
```

Comment:
updated_user_group_model (please differentiate between models and domain objects)

Reply:
Done